testing/go/insert_test.go

```diff
@@ -258,5 +258,50 @@ func TestInsert(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name: "insert with returning",
```

Comment:
Probably should call this `insert returning prepared`, there's an existing test of INSERT RETURNING above. You might consider merging this into it if it's not too much work.